### DISCUSSION TITLE
some Fabric mods didn't work with sinytra connector

### DISCUSSION THREAD

**Discussion by NetherGonnaGiveUp** [Sinytra/Connector#1599] [General]
I used the connector while adding some fabric mods, most mods kinda worked like C2ME but there are some exceptions:

Vulkan
Dynamic Surroundings Fabric

Yes I used latest versions for the mods and I did tested each separately and only pairs with connector mod. I found out that Vulkan and Sinytra Connector won't work and would just outright crashing itself. Perhaps this issue might be fixed? Vulkan mod is worth to support since it's way faster than sodium (at the expense of no shaders) and I was looking out for it to be supported soon via connector.

**Comment by david7919** [comment]
Tried vulkan mod in fcl-launcher with vulkan android libraries mod additionally, doesnt work either.
Gives me a
missing org/lwjgl/vulkan/vkandroidsurfacecreateinfokhr
